docs/api.rst

```diff
@@ -5,5 +5,7 @@ API documentation
     :special-members:
     :members:
 
+.. automodule:: pytket.extensions.qujax._metadata
```

Comment:
Does this automodule usage point to this file?

https://github.com/CQCL/pytket-qujax/blob/7185151efac01d9c5b1ed0f1bf7842a368234ad0/_metadata.py#L2

I don't think there are any public classes or functions in there. Were there some warnings if this was not added in?

Reply:
Yep, exactly. Coverage checks complain if it is not included since it is a python file included in the package, but nothing is printed out by the `.. automodule` directive